sharktank/tests/models/vae/vae_test.py

```diff
@@ -149,7 +149,7 @@ def testVaeIreeVsHuggingFace(self):
             "--iree-opt-outer-dim-concat=true",
             "--iree-llvmgpu-enable-prefetch=true",
             "--iree-hip-waves-per-eu=2",
-            "--iree-dispatch-creation-enable-aggressive-fusion=true",
+            # "--iree-dispatch-creation-enable-aggressive-fusion=true", #TODO: https://github.com/iree-org/iree/issues/20875
```

Comment:
I copied these options from SDXL's config several months ago. It is curious if we really want this one and the rest.

Reply:
Most of these flags are relevant for perf, some maybe be the same as default behavior. 
IMO good have explicit flag config that we want working to catch any regressions.